tf2onnx/optimizer/einsum_optimizer.py

```diff
@@ -373,6 +373,8 @@ def _compute_output_row_expand_dims(self, row, row2=None, ab=False):
         self._check_row_(row, True)
         self._check_arg_('axes', tuple)
         axes = self.kwargs['axes']
+        if not axes:
```

Comment:
This has broken current tests. Please see if this is really required by failed tests. Thanks.